pkg/volume/util.go

```diff
@@ -106,15 +132,20 @@ type recyclerClient interface {
 	CreatePod(pod *api.Pod) (*api.Pod, error)
 	GetPod(name, namespace string) (*api.Pod, error)
 	DeletePod(name, namespace string) error
-	WatchPod(name, namespace string, stopChannel chan struct{}) func() *api.Pod
+	WatchPod(name, namespace string, stopChannel chan struct{}) (<-chan watch.Event, error)
+	Event(eventtype, message string)
```

Comment:
I think it would help if there was godoc in the interface declaration for `WatchPod` and `Event`